### ISSUE TITLE
Build for Ubuntu/Alpine without AVX512 support?

### ISSUE THREAD

**Issue by jurgispods** [lakshayg/tensorflow-build-archived#64]
Hi,

first of all thanks for your effort in creating different flavors of TensorFlow binaries.

I am currently in the process of creating a Docker base image containing somewhat recent versions of Python (>= 3.7) and TensorFlow (>= 1.13) with AVX2 support. You provide a Python 3.7 version which additionally has AVX512F support enabled, but this version won't run on my target system, as the CPU does not support AVX512F.

Is it possible to ship a Linux-targeted version, but _without_ AVX512F support? Ubuntu would be fine, Alpine would be even better for use in Docker containers, although I guess that you could make a Ubuntu-compiled binary work under Alpine as well by swapping musl for libs in the Alpine image.

This seems like a good starting point: https://github.com/better/alpine-tensorflow.

**Comment by f-lng** [comment]
Did you find a wheel for this platform?

**Comment by jurgispods** [comment]
No, I ended up building one from source. Debian/Ubuntu as base image is actually highly recommended, as an Alpine-based image has the problem that many Python packages need to be built from source.
